templates/quiz/results-attempt.php

```diff
@@ -20,18 +21,20 @@
 
 <h2 class="llms-quiz-results-title"><?php printf( __( 'Attempt #%d Results', 'lifterlms' ), $attempt->get( 'attempt' ) ); ?></h2>
 
-<aside class="llms-quiz-results-aside">
-	<?php if ( $attempt->get_count( 'available_points' ) ) : ?>
-		<?php echo llms_get_donut( $attempt->get( 'grade' ), $attempt->l10n( 'status' ), 'default', array( $donut_class ) ); ?>
-	<?php endif; ?>
-	<ul class="llms-quiz-meta-info">
-		<?php if ( $attempt->get_count( 'gradeable_questions' ) ) : ?>
-			<li class="llms-quiz-meta-item"><?php printf( __( 'Correct Answers: %1$d / %2$d', 'lifterlms' ), $attempt->get_count( 'correct_answers' ), $attempt->get_count( 'gradeable_questions' ) ); ?></li>
+<?php if ( ! $attempt->can_be_resumed() ) { ?>
+	<aside class="llms-quiz-results-aside">
+		<?php if ( $attempt->get_count( 'available_points' ) ) : ?>
+			<?php echo llms_get_donut( $attempt->get( 'grade' ), $attempt->l10n( 'status' ), 'default', array( $donut_class ) ); ?>
 		<?php endif; ?>
-		<li class="llms-quiz-meta-item"><?php printf( __( 'Completed: %s', 'lifterlms' ), $attempt->get_date( 'start' ) ); ?></li>
-		<li class="llms-quiz-meta-item"><?php printf( __( 'Total time: %s', 'lifterlms' ), $attempt->get_time() ); ?></li>
-	</ul>
-</aside>
+		<ul class="llms-quiz-meta-info">
+			<?php if ( $attempt->get_count( 'gradeable_questions' ) ) : ?>
+				<li class="llms-quiz-meta-item"><?php printf( __( 'Correct Answers: %1$d / %2$d', 'lifterlms' ), $attempt->get_count( 'correct_answers' ), $attempt->get_count( 'gradeable_questions' ) ); ?></li>
```

Comment:
This is a good occasion to escape all the output that can be escaped :D
can you?

Reply:
done.